exampleSite/static/admin/config.yml

```diff
@@ -307,23 +307,28 @@ collections:
         widget: "object"
         fields:
           - {label: "Discord Name", name: "discord_id", widget: "string"}
-  # encounter metadata
-  - label: "Encounter metadata"
-    name: "encounter-guide"
-    folder: "data/encounter"
-    format: "json"
-    identifier_field: "category"
+  # Encounters Metadata
+  - label: "Encounters"
+    name: "encounter-data"
+    folder: "content/encounters"
+    identifier_field: "title"
     create: true
     fields:
-      - {label: "Category", name: "category", widget: "string"}
-      - label: "Encounters" 
-        name: "encounters"
-        widget: "list"
-        fields:
-          - {label: "Identifier", name: "id", widget: "string"}
-          - {label: "Name of Encounter", name: "name", widget: "string"}
-          - {label: "Image Link", name: "image_link", widget: "string"}
-          - {label: "Encounter Link", name: "encounter_link", widget: "string"}
+      - {label: "Title of Fight", name: "title", widget: "string"}
+      - {label: "Encounter Category", 
+        name: "encounter_category", 
+        widget: "select", 
+        options: ["savage", "ultimate", "extreme"]}
+      - {label: "Card Image", name: "card_image", widget: "image", choose_url: false}
+      - {label: "Tier Image", name: "tier_image", widget: "image", choose_url: false}
+      - {label: "Tier Name (Capitalized)", name: "tier_name", widget: "string"}
```

Comment:
@nonowazu  Question on tier name field. Do you want me to list out all the values and put the in a dropdown so that we get the content team to manually type it in?